_layouts/email.html

```diff
@@ -42,8 +42,13 @@ <h2>{{ page.name }}</h2>
 
 <script type="text/javascript">
     function openEmail() {
+        const isAndroid = /(android)/i.test(navigator.userAgent);
         const subject = encodeURIComponent(`{{ site.default_subject_line }}`.trim());
-        const body = encodeURIComponent(`{{ page.body }}`.trim());
+        const body = encodeURIComponent(
+            isAndroid ?
```

Comment:
Without this check...
🚫 `<br>` shows up on macOS / Gmail
🚫 Double newlines (but no `<br>`) on iOS / Mail
🚫 No newlines and `<br>` shows up on iOS / Gmail

Reply:
Let's go ahead and merge so we can unblock android. Great work!